pokemongo_bot/cell_workers/seen_fort_worker.py

```diff
@@ -31,6 +33,16 @@ def work(self):
         lat = fort['latitude']
         lng = fort['longitude']
 
+        if getattr(self.bot.config, 'forts_show_name', False) == True:
```

Comment:
We don't put default configuration here, only when we parse the configuration. I also happen to think this should be True by default.